Taggers/python/flashggDoubleHTag_cfi.py

```diff
@@ -142,5 +153,38 @@
                        "PhoJetOtherDr := getPhoJetOtherDr()" 
                        ]
                       )
+if flashggDoubleHTag.XYMETCorr_year == 2016:
```

Comment:
@simonepigazzini  Actually the XYMETCorrection code from JME uses 2016, 2017 and 2018 year label and the regression training file which Stepan has shared uses 16, 17, 18. I don't know the reason but if I will use same as XYMET correction year label, it won't work from MJJ training.

Reply:
Considering that it can't be fixed within Mjj regression training file, if you suggest I can try to add label 16, 17 and 18 in JME code here https://github.com/panwarlsweet/flashgg/blob/dev_legacy_runII/Taggers/src/XYMETCorrection.cc

what do you think?

Reply:
@simonepigazzini I tried with above idea but it doesn't accept cms.uint format or cms.int format in the python 
"year := %i"%(flashggDoubleHTag.XYMETCorr_year) ==> I tried with this and got following error:

"TypeError: %d format: a number is required, not uint32"

do you know how to adjust this? Otherwise I think the way we are doing now is good enough.

Reply:
try with:

`year := %i"%(int(flashggDoubleHTag.XYMETCorr_year)) `

Reply:
No still it is not working showing this error:
TypeError: int() argument must be a string or a number, not 'uint32'